src/pages/Examples/TopologyPageSmall/TopologyPageSmall.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import topology from "../../../data/topology.json";
+import { TopologyPageSmallView } from "./TopologyPageSmallView";
+import { TopologyCardSmall } from "../../../components/TopologyCards";
+import { cardStatusBorderTop } from "../../../utils/common";
+
+export const TopologyPageSmall = () => {
+  const cards = topology.map(({ name, status }) => (
+    <TopologyCardSmall
+      key={name}
+      title={name}
+      statusColor={cardStatusBorderTop(status)}
+    />
+  ));
+  return <TopologyPageSmallView cards={cards} />;
```

Comment:
TopologyPageSmall - умный компонент, который содержит контексты, логику формы, стейты, делает запросы и т.д.
всё это передается в глупую вьюшку, которая рендерит что нужно

```suggestion
  return <TopologyPageSmallView topology={topology} />;
```